client/src/Components/Pages/Review/index.js

```diff
@@ -694,7 +694,7 @@ class Review extends Component {
                   >
                     <AgreementLabel
                       htmlFor="agreement"
-                      style={{ pointerEvents: "none" }}
+                      // style={{ pointerEvents: "none" }}
```

Comment:
Hi @thejoefriel, all looks great. just one small note.
this way the checkbox won't toggle when clicking the text. I suggest to uncomment this line and reset that in the <LinkSpan>
```
      style={{ pointerEvents: "auto" }}
```